reason-react-native/src/components/ViewPagerAndroid.re

```diff
@@ -1,22 +1,25 @@
 type element;
 type ref = React.Ref.t(Js.nullable(element));
 
+type onPageScrollArgs =
+  Event.syntheticEvent({
+    .
+    "position": int,
+    "offset": float,
+  });
+
+type onPageSelectedArgs = Event.syntheticEvent({. "position": int});
```

Comment:
For consistency with other components, shouldn't this be named `pageSelectedEvent` and the one above `pageScrollEvent`?